lib/linguist/languages.yml

```diff
@@ -4146,25 +4145,18 @@ Python:
   - ".xpy"
   filenames:
   - ".gclient"
-  - BUCK
-  - BUILD
-  - BUILD.bazel
   - DEPS
   - SConscript
   - SConstruct
   - Snakefile
-  - WORKSPACE
   - wscript
   interpreters:
   - python
   - python2
   - python3
   aliases:
-  - bazel
-  - bzl
-  - rusthon
-  - starlark
```

Comment:
Same comment for the `starlark` alias here.

Reply:
Since the language is moved to a separate one the alias is not needed anymore (I think). Just tried to add it again and there is a failing test: 

```
0.7.7/lib/charlock_holmes/encoding_detector.rb:66: warning: instance variable encoding_list not initialized
/home/runner/work/linguist/linguist/lib/linguist/language.rb:67:in `block in create': Duplicate alias: starlark (ArgumentError)
```

Reply:
Ah, of course! My bad.